rules/macos/persistence_crontab_creation.toml

```diff
@@ -39,7 +39,7 @@ name = "Scheduled Task/Job"
 reference = "https://attack.mitre.org/techniques/T1053/"
 [[rule.threat.technique.subtechnique]]
 id = "T1053.003"
-name = "Cron"
+name = "Scheduled Task/Job: Cron"
```

Comment:
Will be correct when https://github.com/elastic/detection-rules/pull/2073 is merged